docs/release-notes/Percona-Everest-1.1.0-(2024-08-12).md

```diff
@@ -134,10 +134,15 @@ curl -sS "https://raw.githubusercontent.com/percona/everest-doc/main/tools/bin/c
 ```
 
 
-
 **Scenario 2**
 
-If a storage is currently in use by any schedule or backup, you can’t delete it. First, create a new backup or schedule with a different storage. Then, remove the old schedules and backups using the duplicate storage. Once it's no longer in use, you can delete it.
+What to do if you have schedules or backups that are using duplicated storages in different database technologies.
+
+=== ":simple-mongodb: MongoDB :simple-mysql: MySQL"
```

Comment:
Let's add a separator (like `or` for example) between mongo and mysql otherwise it looks weird in the render
```suggestion
=== ":simple-mongodb: MongoDB or :simple-mysql: MySQL"
```